src/Helper.php

```diff
@@ -140,4 +148,38 @@ public static function getKeyDetails($key): array
 
         return $accountDetails;
     }
+
+    /**
+     * Split a two certificate bundle into separate
+     * multi line string certificates
+     * @return array
+     */
+    public static function splitCertificate(string $certificate): array
```

Comment:
i Think that instead of iterating of the lines in search for breakpoints. we can replace this with a regex extraction 
the following regex extracts both the certificated by name.
i think we should replace the certificate splitting with this
```
        preg_match('/^(?<intermediate_certificate>-----BEGIN CERTIFICATE-----.+?-----END CERTIFICATE-----)\n(?<signed_certificate>-----BEGIN CERTIFICATE-----.+?-----END CERTIFICATE-----)$/s',$certificate->getCertificate(),$certificates);
       
        $certificates['intermediate_certificate']
        $certificates['signed_certificate']
```